client/src/components/IconTooltip/index.tsx

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import RightBoat from "../../assets/icons/boat right.png";
+import Volcano from "../../assets/icons/volcano.png";
+import Factory from "../../assets/icons/factory.png";
+import Deciduous from "../../assets/icons/deciduous.png";
+import DeciduousMultiple from "../../assets/icons/deciduousMultiples.png";
+import Evergreen from "../../assets/icons/evergreen.png";
+import EvergreenMultiple from "../../assets/icons/evergreenMultiples.png";
+import Pond from "../../assets/icons/pond.png";
+import PondMultiple from "../../assets/icons/pondMultiples.png";
+import Rock from "../../assets/icons/rock.png";
+import RockMultiple from "../../assets/icons/rockMultiples.png";
+import styles from "./styles.module.css";
+import styled from "styled-components";
+
+interface IconToolTipProps {
+  type:
+    | "string"
+    | "stringMultiple"
+    | "boolean"
+    | "booleanMultiple"
+    | "int"
+    | "intMultiple"
+    | "other"
+    | "otherMultiple"
+    | "import"
+    | "method"
+    | "constructor";
+  data: any;
+}
+
+function getIcon(type: string): any {
```

Comment:
Instead of using `any` as the return type, I think `string` is more accurate as the function returns the file path of an icon.